### PR TITLE
feat: licensed users chart

### PR DESCRIPTION
**Pull request by sjaanus**
Currently showing 2 lines, because backend is not sorting the data.

image

### CODE REVIEW

File: frontend/src/component/insights/components/LineChart/LineChartComponent.tsx [hunk 1]
```diff
@@ -100,16 +105,18 @@ const LineChartComponent: FC<{
...
-        () => ({
-            ...createOptions(
-                theme,
-                locationSettings,
-                setTooltip,
-                Boolean(cover),
-            ),
-            ...overrideOptions,
-        }),
+        () =>
```
Comment (sjaanus) [inline frontend/src/component/insights/components/LineChart/LineChartComponent.tsx:110]: Replaced spreading with deepmerge. That allows me to pinpoint override properties deep in object tree .

File: frontend/src/component/insights/components/LineChart/createChartOptions.ts [hunk 2]
```diff
@@ -3,13 +3,14 @@ import type { ILocationSettings } from 'hooks/useLocationSettings';
...
 import { legendOptions } from './legendOptions';
+import type { ChartOptions } from 'chart.js';
 
 export const createOptions = (
     theme: Theme,
     locationSettings: ILocationSettings,
     setTooltip: React.Dispatch<React.SetStateAction<TooltipState | null>>,
     isPlaceholder?: boolean,
-) =>
+): ChartOptions<'line'> =>
```
Comment (sjaanus) [inline frontend/src/component/insights/components/LineChart/createChartOptions.ts:13]: Fixing type

File: frontend/src/component/insights/components/LineChart/createChartOptions.ts [hunk 3]
```diff
@@ -27,10 +28,6 @@ export const createOptions = (
             tooltip: {
                 enabled: false,
                 position: 'nearest',
-                interaction: {
```
Comment (sjaanus) [inline frontend/src/component/insights/components/LineChart/createChartOptions.ts:30]: Does not exist for liner chart

File: frontend/src/component/insights/components/LineChart/createChartOptions.ts [hunk 4]
```diff
@@ -46,8 +43,6 @@ export const createOptions = (
                 hitRadius: 15,
             },
         },
-        // cubicInterpolationMode: 'monotone',
-        tension: 0.1,
```
Comment (sjaanus) [inline frontend/src/component/insights/components/LineChart/createChartOptions.ts:50]: Does not exist for line chart

**Review by thomasheartman — APPROVED** [review]

**Pull request merged**